### DISCUSSION TITLE
Position Attribute

### DISCUSSION THREAD

**Discussion by prairiesnpr** [PiotrMachowski/lovelace-xiaomi-vacuum-map-card#408] [Q&A] [answered]
I'm sorry if I missed this, but what is the attribute and format that this card looks for to map the vacuum position?  I'm trying to use it with a template vacuum.

**Comment by PiotrMachowski** [comment, marked as answer]
I'm not sure what you mean, do you mean `status`?

**Reply by prairiesnpr** [reply]
Where does the card read the x, y coordinates from to plot the position of the robot on the map?  And what format is required for those coordinates? My assumption was that it would be an attribute on the vacuum entity,  but none of the attribute names and coordinate formats I've tried have resulted in the card plotting a location.

**Reply by PiotrMachowski** [reply]
Oh, no, map rendering is done in map extractor, not in the card. Card just displays a map image

**Reply by prairiesnpr** [reply]
Ouch, I completely missed that, back to square one then.  Thanks, card does look great.

**Reply by PiotrMachowski** [reply]
Out of curiosity, what do you want to achieve?

**Reply by prairiesnpr** [reply]
I have a Husqvarna automower that reports it's position via GPS.  It creates a device tracker which can then be displayed using the map card, but that shows you an icon in a black square since there is little detail on this maps. 
![SmartSelect_20220605-074835_Home Assistant](https://user-images.githubusercontent.com/18637008/172049050-8c5e840d-2f1f-45d0-a9cc-69bd5bcf4ee9.jpg)
  I looked at your card and it has the ability to show an image, so I was going to use a satellite image of my house then translate the GPS coordinates to pixel coordinates so that the mowers position would be displayed on top of the satellite image. Something like this. 
![SmartSelect_20220605-074701_Automower Connect](https://user-images.githubusercontent.com/18637008/172049026-54633b47-8f32-4612-9bcc-e35ad123b1f2.jpg)

**Reply by PiotrMachowski** [reply]
Unfortunately it's not possible at this moment :(
